package io.maksymuimanov.task.exception;

public class CacheManagingException extends RuntimeException {
    public CacheManagingException(Throwable cause) {
        super(cause);
    }
}
